### ISSUE TITLE
Missing alignment of messages inside Script Debug section of the SMB-tab

### ISSUE THREAD

**Issue by ga-zelle** [nightscout/AndroidAPS#2058]
All messages from determine-basal.js are patched together end-to-end without any line breaks. This started in 3.0 dev versions, was (nearly) fixed in 3.0.0.1 and now happens again in 3.1.0.2.

**Comment by Andries-Smit** [comment]
@ga-zelle could you please add screenshots?

**Comment by ga-zelle** [comment]
This is what it looked like in version 2 .8
![grafik](https://user-images.githubusercontent.com/50909826/188269049-c303a576-3ebb-4aa4-8a30-6b7c41975e0b.png)

Here is a screenshot from 3.0.0.1
![Screenshot_20220303-000826_AndroidAPS](https://user-images.githubusercontent.com/50909826/188269153-e41bd616-0a7a-4766-8595-3f2ef26884b7.jpg)

And this one was a minute ago in 3.1.0.2:
![Screenshot_20220903-135138_AndroidAPS](https://user-images.githubusercontent.com/50909826/188269313-f7f72c8c-13d3-4b44-98c9-6256866f62ae.jpg)

By the way, in the logfile those messages are nicely separated as usual.

**Comment by Andries-Smit** [comment]
Looks like semi column was replace by a new line before. Could be fixed easy. There are some semi columns missing in the debug string... Will check that too

**Comment by ga-zelle** [comment]
Interesting; I never inserted newline or semcolon for my extra debug messages before.

**Comment by Andries-Smit** [comment]
You are right, guess this might be handled a bit different after the conversion to Kotlin.
https://github.com/nightscout/AndroidAPS/blob/master/app/src/main/java/info/nightscout/androidaps/plugins/aps/logger/LoggerCallback.kt#L22
I expected that the next line would do the trick
``` kt
logBuffer.appendLine(obj1.toString())
```
But it did not :( 
Might @MilosKozak know what the the real fix is.

**Comment by ga-zelle** [comment]
I am pretty sure it was nearly solved in 3.0.0.1, apart from the first few lines. I happen to have a screen shot from that version:
![Screenshot_20220727-040343_AndroidAPS](https://user-images.githubusercontent.com/50909826/188291638-6cee4a37-660f-4266-a251-0bcfe6ca6ffc.jpg)

**Comment by justmara** [comment]
@ga-zelle [this line](https://github.com/nightscout/AndroidAPS/blob/master/app/src/main/java/info/nightscout/androidaps/plugins/aps/openAPSSMB/OpenAPSSMBFragment.kt#L140) removes line-breaks. and it is replicated over all plugins for now already :)

**Comment by ga-zelle** [comment]
Thank you @justmara for digging so deep. I 'll try dropping the Replace part, not the whole line.

**Comment by olorinmaia** [comment]
This has been fixed/improved in dev (AAPS 4.0.0) closing issue
